api/handlers.go

```diff
@@ -706,3 +726,18 @@ func max(x, y uint64) uint64 {
 	}
 	return y
 }
+
+// isSpecialAccount returns true if addr belongs to a special account, false otherwise.
+// The function returns an error in case it fails to retrieve the special accounts list.
+func (si *ServerImplementation) isSpecialAccount(addr string) (bool, error) {
+	// Special accounts non handling
+	sa, err := si.db.GetSpecialAccounts()
```

Comment:
probably worth caching this in a field of ServerImplementation so we don't hit the db every time.

Reply:
I thought about it but the db implementation already caches it, should we cash it twice?

Reply:
Oh oops, yeah postgres.go caching is enough